tckprocess.adoc

```diff
@@ -0,0 +1,406 @@
+
+-   Name and version of NoSQL implementations used in persistence tests
+
+[#_public_tck_results_summary]
+### Public TCK Results Summary
+
+While certification is on your honor, the community MUST be able to see
+your test results summary. At a minimum a results summary MUST:
+
+-   Be publicly visible with no password protection or sign-up
+
+-   Include a Summary Page containing:
+
+    -   All information in the above Certification Request
+
+    -   The Total number of tests run and passed. If you are running a
+        platform TCK, this means reporting the tests run for each
+        required TCK, which includes the platform TCK and some of the
+        individual TCKs that are not integrated with the platform TCK.
+
+An optional "Test List Page" showing all tests run may be linked from
+the Summary Page. The Summary Page URL is the URL that MUST be included
+in any Certification Requests.
+
+The following are explicitly not requirements:
+
+-   The Ability for the public to run the tests themselves
+
+-   Full log output of the TCK
+
+Implementors may supply this information and provide support for how to
+run a TCK against their implementation, but it is not required.
+
+[#_certification_resolution]
+### Certification Resolution
+
+Approval that the TCK requirements have been met is a prerequisite for getting
+the corresponding runtime listed as complible implemenations of a particular MicroProfile platform release. The required approval processes is:
+
+-   Approval by lazy consensus after a period of two weeks (14 days)
+
+-   Approval by a majority vote of the specification contributors as soon as
+    it happens.
+
+    -   The sum of the +1/-1 votes must be greater than 50% of the
+        specification committers.
+
+All specification contributors are encouraged to review the request
+and associated supporting materials. Reviewers of a certification
+request MUST carefully check the validity of all required data, in
+particular:
+
+-   the data is complete
+
+-   the number of tests passed is consistent with the first
+    implementation used to validate the TCK
+
+-   TCK version and digital fingerprint match.
+
+-   test results are public and do not require special signup or viewing
+    steps
+
+Any committer on the specification may vote against the
+certification request on the basis that the clearly defined requirements
+of the TCK process have not been met. This means that if there is a (-1)
+vote, lazy consensus is no longer an option and a majority vote MUST
+take place.
+
+[#_accepted_certification_requests]
+#### Accepted Certification Requests
+
+Certification requests that are reviewed and found to meet the
+requirements will be marked accepted by closing an issue with an
+`accepted` label. A pointer/link to the issue MUST then be emailed to
+mailto:tck@eclipse.org[tck@eclipse.org], as required by the
+https://www.eclipse.org/legal/tck.php[Eclipse Foundation Technology Compatibility Kit License]. 
```

Comment:
We should mention that this email needs to be sent by the testing party. Either when closing the issue or in this TCK process.

Reply:
Ideally, if we could automate closing of the certification request with something that will trigger the addition of implementation to the website which would also contain sending of this email so the testing party (implementation) doesn't need to care about the process after the certification issue is filled I think that would be preferable. Not sure if we can somehow automate this with marketing tasks (website updates)?

Reply:
I think it can be done either manualy or automatically. I would rather not spec it as it is implementation details.

Reply:
I have added explicitly who should send the email to EF.